docker/.env

```
@@ -239,6 +239,11 @@ SMQ_USERS_HTTP_HOST=users
 SMQ_USERS_HTTP_PORT=9002
 SMQ_USERS_HTTP_SERVER_CERT=
 SMQ_USERS_HTTP_SERVER_KEY=
+SMQ_USERS_GRPC_HOST=users
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMQ_USERS_GRPC_HOST key should go before the SMQ_USERS_HTTP_HOST key

(UnorderedKey)